### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by xmatmen**
* Add the poll stuff (https://i.imgur.com/hcRLAUn.png)
* Change updater license to GPL-3.0
* Make run script support multiple subscripts
* Make the run script prompt the user for a GitHub token if none is set
* Only check PRs that are at least 10h old and haven't been edited in the last hour
* Only close PRs that haven't been updated in the last 24h
* Don't check WIP PRs

### CODE REVIEW

File: updater/updater.js [hunk 1]
```diff
@@ -32,6 +32,13 @@ class Updater {
 		let changed = false;
 
 		for (const pullRequest of pullRequests.data) {
+			if (/^(\[|\()?WIP:?(\]|\))?|(\[|\()WIP(\]|\))$/i.test(pullRequest.title)) continue; // Don't check the PR if it's WIP
+
+			const createdAt = Date.parse(pullRequest.created_at).valueOf();
+			const updatedAt = Date.parse(pullRequest.updated_at).valueOf();
+			if (Date.now() - createdAt < 10 * 60 * 60 * 1000) continue; // Don't check the PR if its not older than 10h
```
Comment (xmatmen) [inline updater/updater.js]: Not sure if this should actually be this high, i think 6h is enough

File: updater/updater.js [hunk 2]
```diff
@@ -32,6 +32,13 @@ class Updater {
 		let changed = false;
 
 		for (const pullRequest of pullRequests.data) {
+			if (/^(\[|\()?WIP:?(\]|\))?|(\[|\()WIP(\]|\))$/i.test(pullRequest.title)) continue; // Don't check the PR if it's WIP
```
Comment (xmatmen) [inline updater/updater.js]: Not sure if this regex matches everything

**Comment by 183k** [comment]
yes

File: updater/updater.js (hunk 2 above)
Comment (183k) [reply updater/updater.js]: yes

File: run.sh [hunk 3]
```diff
@@ -5,7 +5,21 @@ while true; do
 		git pull origin master	# Make sure the local copy is up-to-date
 		./migrate.sh			#
 
-		./updater/run.sh
+		subscripts=()
+		
+		if [ -z "$discordToken" ]							# Make sure the githubToken env var is set
```
Comment (Confuseh) [inline run.sh]: Uh this seems wrong

Comment (xmatmen) [reply run.sh]: How so?

Comment (Confuseh) [reply run.sh]: That one says $discordToken and lines underneath have $githubToken

Comment (xmatmen) [reply run.sh]: Oh. I see

Comment (Confuseh) [reply run.sh]: Well, fix it.

**Comment by xmatmen** [comment]
will do

Am 16.10.2017 2:07 PM schrieb "Confuseh" <notifications@github.com>:

File: run.sh (hunk 3 above)
Comment (xmatmen) [reply run.sh]: will do

**Comment by xmatmen** [comment]
no u @adk8674

**Comment by NotSoChaos** [comment]
Merged.
Approval rate: 3/12 (25%)

**Comment by level6ascend** [comment]
fuck you

**Comment by xmatmen** [comment]
thats rude

**Comment by level6ascend** [comment]
no fuck you seriously this bot is fucking dead

**Comment by level6ascend** [comment]
maybe make it alive or something so people can...like...**USE** it

**Comment by xmatmen** [comment]
well the point of the project is that everyone can add stuff to it, not that its necessarily useful

**Comment by level6ascend** [comment]
but it's offline.

**Comment by xmatmen** [comment]
wdym its offline

**Comment by level6ascend** [comment]
the bot is fucking offline can't you fucking see you fat piece of shit

**Comment by xmatmen** [comment]
hey, that's rude. also, what bot. if you're talking about the discord bot, it's not related to this project

**Comment by level6ascend** [comment]
yeah, the discord bot

**Comment by xmatmen** [comment]
as i said, it's not related to this project

**Pull request merged**